kuma/wiki/templates/wiki/document.html

```diff
@@ -424,8 +424,8 @@ <h2 class="offscreen">{{ _('Document Tags and Contributors') }}</h2>
     </div> <!-- ends "main-content" -->
 
   <menu type="context" id="edit-history-menu">
-    <menuitem data-action="$edit" label="{{_('Edit page')}}"></menuitem>
-    <menuitem data-action="$history" label="{{_('View page history')}}"></menuitem>
+    <menuitem data-action="{{ url('wiki.edit_document', document.full_path, locale=document.locale) }}" label="{{_('Edit page')}}"></menuitem>
+    <menuitem data-action="{{ url('wiki.document_revisions', document.full_path) }}" label="{{_('View page history')}}"></menuitem>
```

Comment:
I'm using the same URL for the `page-buttons` revision page, so I cannot provide a given reason, but since we've not had bugs about the other URL, this shouldn't be an issue either.  Ask @jezdez maybe?